components/post_edit_history/edited_post_item/__snapshots__/edited_post_item.test.tsx.snap

```diff
@@ -0,0 +1,273 @@
+            className="hidden-xs"
+            id="editPostRestoreTooltip"
+          >
+            Restore this version
+          </Tooltip>
+        }
+        placement="left"
+        trigger={
+          Array [
+            "hover",
+            "focus",
+          ]
+        }
+      >
+        <ForwardRef
+          aria-label="Select to restore an old message."
+          className="edit-post-history__icon__button restore-icon"
+          compact={true}
+          icon="refresh"
+          onClick={[Function]}
+          size="sm"
+        />
+      </OverlayTrigger>
+    </div>
+  </PostAriaLabelDiv>
+</div>
+`;
+
+exports[`components/post_edit_history/edited_post_item should match snapshot when isCurrent is true 1`] = `
+<div
+  className="edit-post-history__container edit-post-history__container__background"
+  onClick={[Function]}
+>
+  <PostAriaLabelDiv
+    className="a11y__section post"
+    id="searchResult_post_id"
+    post={
+      Object {
+        "channel_id": "",
+        "create_at": 0,
+        "delete_at": 0,
+        "edit_at": 0,
+        "hashtags": "",
+        "id": "post_id",
+        "is_pinned": false,
+        "message": "post message",
+        "metadata": Object {
+          "embeds": Array [],
+          "emojis": Array [],
+          "files": Array [],
+          "images": Object {},
+          "reactions": Array [],
+        },
+        "original_id": "",
+        "pending_post_id": "",
+        "props": Object {},
+        "reply_count": 0,
+        "root_id": "",
+        "type": "system_add_remove",
+        "update_at": 0,
+        "user_id": "user_id",
+      }
+    }
+  >
+    <div
+      aria-hidden="true"
+      className="edit-post-history__title__container"
+    >
+      <div
+        className="edit-post-history__date__badge__container"
+      >
+        <ForwardRef
+          aria-label="Toggle to see an old message."
+          className="edit-post-history__icon__button"
+          compact={true}
+          icon="chevron-down"
+          size="sm"
+        />
+        <span
+          className="edit-post-history__date"
+        >
+          <Connect(injectIntl(Timestamp))
+            ranges={
+              Array [
+                Object {
+                  "display": <Memo(MemoizedFormattedMessage)
+                    defaultMessage="Today"
+                    id="date_separator.today"
+                  />,
+                  "equals": Array [
+                    "day",
+                    0,
+                  ],
+                },
+                Object {
+                  "display": <Memo(MemoizedFormattedMessage)
+                    defaultMessage="Yesterday"
+                    id="date_separator.yesterday"
+                  />,
+                  "equals": Array [
+                    "day",
+                    -1,
+                  ],
+                },
+              ]
+            }
+            value={0}
+          />
+        </span>
+        <div
```

Comment:
@michelengelen I think this should be using the `Tag` component (formerly) `Badge`. Is your work on this merged yet?